test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -8,13 +8,151 @@ var chai = require('chai')
   , DataTypes = require(__dirname + '/../../../../lib/data-types')
   , _ = require('lodash');
 
+
 if (dialect.match(/^postgres/)) {
   describe('[POSTGRES Specific] QueryInterface', function () {
     beforeEach(function () {
       this.sequelize.options.quoteIdenifiers = true;
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+    describe('createFunction', function () {
+
+      beforeEach( function () {
```

Comment:
if the test suite fails or stops unexpectedly causing afterEach's cleanup to not be called and the test suite is then restarted the cleanup in beforeEach will be needed.  What do you suggest?